content/ja/docs/concepts/workloads/controllers/replicaset.md

```diff
@@ -1,8 +1,15 @@
 ---
 reviewers:
 title: ReplicaSet
+feature:
+  title: セルフヒーリング
+  anchor: ReplicaSetがどのように動くか {#how-a-replicaset-works}
+  description: >
+    起動に失敗したコンテナを再起動し、ノードが停止したときにコンテナを置き換えて再スケジュールし、
+    ユーザーが定義したヘルスチェックに応答しないコンテナを停止させ、サービスの準備が出来るまではそれらのコンテナをクライアントに示しません。
 content_type: concept
-weight: 20
+weight: 30
```

Comment:
This value is 20 on [the content of `en`](https://raw.githubusercontent.com/kubernetes/website/523b67d55ecfdcf5c86b0707bcd3fc1207fd7fb8/content/en/docs/concepts/workloads/controllers/replicaset.md).
What is the reason for updating this value with `30`?